### ISSUE TITLE
Backport: Rancher server has TLS 1.0 enabled 

### ISSUE THREAD

**Issue by alena1108** [rancher/rancher#18177]
Backport https://github.com/rancher/rancher/issues/15885

**Comment by wlcumt** [comment]
建议尽快切换到TLS1.2上去，TLS1.0业界已经在停止使用了。

**Comment by alena1108** [comment]
Fix is available in rancher `v2.1.7-rc3`

**Comment by soumyalj** [comment]
Tested with rancher v2.1.7-rc3
Created a rancher server with certs, configured a DNS name pointing to the rancher server. 
Verified that TLS1 and TLS1_1 are not supported.
```
~# openssl s_client -connect sabc.com:443 -tls1
CONNECTED(00000003)
140270436148888:error:1409442E:SSL routines:ssl3_read_bytes:tlsv1 alert protocol version:s3_pkt.c:1487:SSL alert number 70
140270436148888:error:1409E0E5:SSL routines:ssl3_write_bytes:ssl handshake failure:s3_pkt.c:656:
---
no peer certificate available
---
No client certificate CA names sent

```
```
openssl s_client -connect sabc.com:443 -tls1_1
CONNECTED(00000003)
139815907042968:error:1408F10B:SSL routines:SSL3_GET_RECORD:wrong version number:s3_pkt.c:362:
---
no peer certificate available
---
No client certificate CA names sent
---
SSL handshake has read 5 bytes and written 0 bytes
---
```
Verified that TLS1.2 is enabled.
```
 openssl s_client -connect sabc.com:443 -tls1_2
CONNECTED(00000003)
depth=2 O = Digital Signature Trust Co., CN = DST Root CA X3
verify return:1
depth=1 C = US, O = Let's Encrypt, CN = Let's Encrypt Authority X3
verify return:1
depth=0 CN = *.qa.rancher.space
verify return:1
---
Certificate chain
 0 s:/CN=*.qa.rancher.space
   i:/C=US/O=Let's Encrypt/CN=Let's Encrypt Authority X3
 1 s:/C=US/O=Let's Encrypt/CN=Let's Encrypt Authority X3
   i:/O=Digital Signature Trust Co./CN=DST Root CA X3
---

```

Using : * Using https://www.ssllabs.com/ssltest/index.html
![screen shot 2019-02-25 at 2 25 31 pm](https://user-images.githubusercontent.com/18536626/53373052-511f3780-3909-11e9-8245-474bc7d45365.png)

Using nmap
```
~# nmap --script ssl-enum-ciphers -p 443 sabc.com

Starting Nmap 7.01 ( https://nmap.org ) at 2019-02-25 22:28 UTC
Nmap scan report for soumyanewtest.qa.rancher.space (134.209.6.67)
Host is up (0.0014s latency).
PORT    STATE SERVICE
443/tcp open  https
| ssl-enum-ciphers: 
|   TLSv1.2: 
|     ciphers: 
|       TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 (secp256r1) - A
|       TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384 (secp256r1) - A
|     compressors: 
|       NULL
|     cipher preference: server
|_  least strength: A

Nmap done: 1 IP address (1 host up) scanned in 0.81 seconds




```
